### PR TITLE
ceph-disk: ceph-disk on FreeBSD should not use mpath-code

### PR DESCRIPTION
**Pull request by wjwithagen**
Signed-off-by: Willem Jan Withagen <wjw@digiware.nl>

### CODE REVIEW

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-04-28_19:45:54-rados-wip-sage-testing---basic-smithi/

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-04-29_00:40:18-ceph-disk-wip-sage-testing---basic-mira/

File: src/ceph-disk/ceph_disk/main.py [hunk 1]
```diff
@@ -4228,19 +4241,29 @@ def get_oneliner(base, name):
...
-        return fstype
+    if FREEBSD:
+        fstype, _, ret = command(
+            [
+                'fstyp',
+                '-u',
+                dev,
+            ],
+        )
+        if ret:
```
Comment (tchaikov) [inline src/ceph-disk/ceph_disk/main.py]: > EXIT STATUS
>     The fstyp utility exits 0 on success, and >0 if an error occurs or the filesystem type is not recognized.
see http://www.unix.com/man-page/freebsd/8/fstyp/, 

so should be `if ret == 0`, i guess?

**Comment by wjwithagen** [comment]
On 29-04-17 02:42, Sage Weil wrote:
> http://pulpito.ceph.com/sage-2017-04-29_00:40:18-ceph-disk-wip-sage-testing---basic-mira/

I'm having trouble interpreting the results there.

Looks like the 2 wrong tests too 5 and 1 hour to complete and the
correct test only 28 min.
And I only see not installed warnings...

Could it be that the faulty tests did not correctly start to begin with?

--WjW

**Comment by tchaikov** [comment]
@wjwithagen i reran the ceph-disk suite at http://pulpito.ceph.com/kchai-2017-04-29_03:01:51-ceph-disk-wip-sage-testing---basic-vps/. and it passed.

**Pull request merged**